indico/modules/designer/pdf.py

```diff
@@ -68,6 +68,8 @@ def __init__(self, template, config):
         self.config = self._build_config(config)
         self.template = template
         self.tpl_data = self._process_tpl_data(template.data)
+        if template.backside_template:
+            self.backside_tpl_data = self._process_tpl_data(template.backside_template.data)
```

Comment:
It looks like you only access `backside_tpl_data` for the foldable version so not having it if there's no backside won't break anything since you cannot create a single-sided foldable badge, but I'd still set the attribute to `None` if there is no backside template for the sake of always having the attribute present.